### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by zerebubuth**
A few tests were failing against the new CI tests:

1. #418 WOF tests needed fixture data with the l10n column to pass.
2. #465 Changes to `fitness` had excluded `amenity=gym` and `fitness_station`.
3. #566, #663 Sort key changes had resulted in a bunch of changes.
4. #647 OSM way 17125595 was part of an ICN network, now changed to NCN, and apparently the subject of [much discussion](https://www.openstreetmap.org/changeset/39469430) as to whether it should be a *CN or MTB route. Instead, I've included an unclassified way which is part of a stable ICN.

@rmarianski could you review, please?

### CODE REVIEW

File: test/418-wof-l10n_name.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
       'name:spa': type(None) })
 
 # San Francisco (osm city)
-# https://whosonfirst.mapzen.com/data/858/826/41/85882641.geojson
+# http://www.openstreetmap.org/node/26819236
```
Comment (rmarianski) [inline test/418-wof-l10n_name.py]: Whoops! Sorry forgot to update this.

File: yaml/pois.yaml [hunk 2]
```diff
@@ -163,9 +163,15 @@ filters:
...
-      leisure: [fitness_centre, gym, fitness_station]
+      any:
+        leisure: [fitness_centre, gym]
+        amenity: gym
     min_zoom: LEAST(zoom + 3.98, 17)
     output: {kind: fitness}
+  - filter:
+      leisure: fitness_station
+    min_zoom: LEAST(zoom + 3.98, 17)
+    output: {kind: fitness_station}
```
Comment (rmarianski) [inline yaml/pois.yaml]: We don't want `fitness_station` to normalize to `kind=fitness`?

**Comment by rmarianski** [comment]
:+1:

File: yaml/pois.yaml (hunk 2 above)
Comment (zerebubuth) [inline yaml/pois.yaml]: I don't think so... at least, that's not what the tests said!

There's a big difference between a gym and a "push up" station in the park, so I think it makes sense to keep them separate.

**Pull request merged**